File: server/groute/src/main/java/com/ssafy/groute/dto/PlanReview.java
package com.ssafy.groute.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class PlanReview {
    private int id;
    private String userId;
    private int planId;
    private String title;
    private String content;
    private double rate;
    private String img;
    private String createDate;
}

File: client/Groute/app/src/main/java/com/ssafy/groute/src/dto/PlanReview.kt
package com.ssafy.groute.src.dto

data class PlanReview(
    val content: String,
    val createDate: String,
    val id: Int,
    val img: String,
    val planId: Int,
    val rate: Double,
    val title: String,
    val userId: String
){
    constructor(planId: Int,userId: String,content: String,rate: Double,img: String) : this(content,"",0,img,planId, rate,"",userId)
    constructor(planId: Int,userId: String,content: String,rate: Double,img: String,id:Int): this(content,"",id,img,planId,rate,"",userId)
}